assets/css/app.scss

```diff
@@ -7,6 +7,11 @@
 @import "bulma";
 @import "bulma-extensions";
 
+$rism-navbar-height: 52px;
```

Comment:
These customizations should probably be moved to `_sass/rism-theme.scss`, and the app.scss deprecated for custom overrides. That way we can better control the order of overrides, and making it so that local sites can override variables. Since the `app.scss` styles are processed last, they cannot be overriden by the `site-theme.scss` files.

Reply:
I can make and test that change, if you agree.